tasks/whitelist-env.yml

```diff
@@ -0,0 +1,25 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: eu.gcr.io/census-gcr-rm/rm/census-rm-toolbox
+params:
+  GCP_PROJECT_NAME:
+  ADMIN_SERVICE_ACCOUNT_JSON:
+  KUBERNETES_CLUSTER:
+inputs:
+  - name: census-rm-whitelist
+run:
+  path: bash
+  args:
+    - -exc
+    - |
+      export GOOGLE_APPLICATION_CREDENTIALS=/home/toolbox/gcloud-service-key.json
+      export GCP_PROJECT=$GCP_PROJECT_NAME
+      cat >$GOOGLE_APPLICATION_CREDENTIALS <<EOL
+      $ADMIN_SERVICE_ACCOUNT_JSON
+      EOL
+      gcloud auth activate-service-account --key-file $GOOGLE_APPLICATION_CREDENTIALS
+      gcloud container clusters get-credentials ${KUBERNETES_CLUSTER} --zone europe-west2 --project ${GCP_PROJECT}
+      cd cloudshell_utilities
```

Comment:
I don't know if that'll be enough because you're not cd'ing into the toolbox repo.  Only 1 way to find out.

Reply:
I think it will start in the home directory when it connects, by default